geomesa-fs/geomesa-fs-tools/src/main/scala/org/locationtech/geomesa/fs/tools/FsDataStoreCommand.scala

```diff
@@ -43,6 +45,17 @@ trait FsDataStoreCommand extends DataStoreCommand[FileSystemDataStore] {
   }
 }
 
+object FsDataStoreCommand {
+  val facSet = new AtomicBoolean(false)
```

Comment:
[minor] you don't need an atomic boolean here if you're accessing it in a synchronized block